schemas/content/product.schema.json

```diff
@@ -60,81 +53,68 @@
           "description":
             "The internal unique ID of the product in the commerce backend system."
         },
-        "xdm:masterProductSKU": {
-          "title": "Master Product SKU",
-          "type": "string",
+        "xdm:prices": {
```

Comment:
Does the price structure also need a GEO property?

Reply:
Maybe, but then we instead of GEO we should be more generic and add "channel", "scope" or "context".  Prices could be per region, channel, customer group, location etc.

Reply:
Maybe we introduce an `xdm:condition` property, which holds multiple properties. Each property equals the XDM property name that is being compared. The value of the property would be (for now) `xdm:equals`.

Resulting in something like this:

```json
"xdm:condition": {
  "xdm:country": {
    "xdm:equals": "DE"
  },
  "xdm:channel.@type": {
    "xdm:equals": "https://ns.adobe.com/xdm/channel-types/web"
  }
}
```

Reply:
I think we can go with the simple approach for now, without conditions. But I like the conditional properties, they should be standardized. I created #362 for that.